template/README.md

```diff
@@ -70,6 +70,14 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 
 ## How To...
 
+### Install a Dependency
+
+The generated project only includes *build* dependencies. Project-specific dependencies (Redux, React Router, etc.) must be installed separately. These dependencies can be installed with `npm`, without having to modify any existing configuration.
```

Comment:
> The generated project only includes *build* dependencies.

Because `npm install --save <library-name>` is considered a build dependency too the above sentence looks confusing to me.

I would like to suggest a slighty change:

> The generated project only includes React and ReactDOM as dependencies. Project-specific dependencies (Redux, React Router, etc.) can be installed separately. These dependencies can be installed with `npm`, without having to modify any existing configuration.